integrations/external_ip.go

```diff
@@ -0,0 +1,22 @@
+package integrations
+
+import (
+	"fmt"
+	"net"
+)
+
+const NoExtIPApi = "no external ip address API available"
```

Comment:
Codacy found an issue: [exported const NoExtIPApi should have comment or be unexported](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)